fixtures/watcher/watcher.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Watcher Fixtures</title>
+  </head>
+  <body>
+    <main>
+      <h1>Watcher Test Cases</h1>
+
+      <div style="margin-bottom: 5em">
+        <h2>Test Case 1 - Button Click</h2>
+
+        <div id="color-block">This isn't readable is it...</div>
+        <button onclick="updateColour()">Click me</button>
+
+        <style>
+          #color-block {
+            background-color: white;
+            color: white;
+            width: 100px;
+            height: 100px;
+            margin-bottom: 1em;
+            display: none;
+          }
+        </style>
+
+        <script>
+          function updateColour() {
+            const colorBlock = document.getElementById("color-block");
+            colorBlock.style.backgroundColor = "gray";
+            colorBlock.style.display = "block";
+
+            document.title = "Watcher - Test Case 1 - Button Click";
+          }
+        </script>
+      </div>
```

Comment:
Decided to put `script` and `style` tags within each `div` "test block". Can merge them all necessary, this reads easier but no strong opinion here.